test/parallel/test-readline-keys.js

```diff
@@ -44,6 +44,39 @@ function addTest(sequences, expectedKeys) {
   assert.deepStrictEqual(keys, expectedKeys);
 }
 
+const addKeyIntervalTest = (sequences, expectedKeys, interval = 150,
+                            assertDelay = 150) =>
```

Comment:
I think that would be more clear but named functions could be better as it's hard to tell what is going on here.